File: API/mail.php
<?php     
require_once('PHPMailer/PHPMailerAutoload.php');

$mail = new PHPMailer();
$mail->isSMTP();
$mail->SMTPAuth = true;
$mail->$SMTPSecure = 'ssl';
$mail->Host = 'smtp.gmail.com';
$mail->Port = '465';
$mail->isHTML();
$mail->Username = '360storereport@gmail.com';
$mail->Password = 'Abbasali@360store';
$mail->SetFrom('shahbazlatheef@gmail.com');
$mail->Subject = 'Hello shaz';
$mail->Body = "Hi Shaz";
$mail->AddAddress('shahbazlatheef@gmail.com');
$mail->Send();
?>

File: API/mail.py
import smtplib
sender = '360storereport@gmail.com'
recipient = 'shahbazlatheef@gmail.com'
content = "hello world"
username = '360storereport@gmail.com'
password = 'Abbasali@360store'
mail = smtplib.SMTP('smtp.gmail.com', 587)
mail.ehlo()
mail.starttls()
mail.login(username, password)
header = 'To:' + recipient + '\n' + 'From: ' \
    + sender + '\n' + 'Subject:testing \n'
content = header+content
mail.sendmail(sender, recipient, content)
mail.close()
